python/sqlflow_submitter/tensorflow/train.py

```diff
@@ -182,16 +181,12 @@ def train(is_keras_model,
           pai_table="",
           pai_val_table=""):
     assert validate_select != ""
-    assert verbose >=0 and verbose <= 3
-    if is_keras_model:
-        if verbose == 1:
-            tf.get_logger().setLevel((4-verbose) * 10)  # logging.INFO levels range from 10~40
-    else:
-        if TF_VERSION_2:
-                tf.get_logger().setLevel((4-verbose) * 10)
-        else:
-            if verbose >= 2:
-                tf.logging.set_verbosity(tf.logging.INFO)
+    assert 0 <= verbose <= 3
+    is_keras_model = not issubclass(estimator, (tf.estimator.Estimator, tf.estimator.BoostedTreesEstimator))
```

Comment:
Should call it `is_premade_estimator`.  A model is Keras model if it's not an `Estimator` does not make sense, even though we only support Keras and Estimators

Reply:
`is_estimator` is better, there may be some models in `sqlflow_models` that implement the estimator interface